android-sdk/src/main/java/com/checkout/android_sdk/FramesLogger.kt

```diff
@@ -120,6 +120,15 @@ internal class FramesLogger {
         )
     }
 
+    fun sendCheckoutApiClientInitialisedEvent() {
```

Comment:
Can you please check the naming convention for other methods for logging ?, Not sure if `sendCheckoutApiClientInitialisedEvent` send prefix sounds great, instead using what is used elsewhere or may be `logCheckoutApiClientInitialised`

Reply:
Sure. I have included naming conventions from the existing code. For instance, for sending a payment form presented event existing code has a name as "sendPaymentFormPresentedEvent" rather than logPaymentFormPresentedEvent. Shall we follow the existing naming convention or prefer for the current change more suitable naming conventions like log event type?
<img width="602" alt="code" src="https://user-images.githubusercontent.com/102042908/163136798-cb794c94-32e7-4ec3-ad48-a601b868acec.png">